content/04.network_applications.md

```diff
@@ -1,16 +1,19 @@
-In many cases, solutions rely on representing knowledges graphs in a low dimensional space, which is a process called representation learning.
-This space preserves a knowledge graph's local and/or global structure and can support efforts to apply machine learning methods to make predictions.
-In the next sections we review the unifying techniques that construct this low dimensional space and unifying applications that use this space to solve biomedical problems.
+Knowledge graphs can help researchers tackle many biomedical problems such as finding new treatments for existing drugs [@doi:10.7554/eLife.26726], aiding efforts to diagnose patients [@arxiv:1611.07012] and identfying associations between diseases and biomolecules [@doi:10.1155/2017/2498957].
+In many cases, solutions rely on representing knowledge graphs in a low dimensional space, which is a process called representational learning.
+This goal of this process is to retain and encode the local and/or global structure of a knowledge graph that are relevant to the problem while transforming the graph into a representation that can be readily used with machine learning methods to build predictors
+In the following sections we review methods that construct this low dimensional space (Unifying Representational Learning Techniques) and discuss applications that use this space to solve biomedical problems (Unifying Applications).
 
-### Unifying Techniques
+### Unifying Representational Learning Techniques
 
 Mapping high dimensional data into a low dimensional space greatly improves modeling performance in fields such as natural language processing [@arxiv:1301.3781; @arxiv:1310.4546] and image analysis [@arxiv:1512.03385].
-The success of these approaches provides rationale for representing knowledge graphs into a low dimensional space [@arxiv:1709.05584].
-Techniques that construct this representation often require information on how nodes are connected with one another [@raw:gongreplearn; @doi:10.1109/TKDE.2016.2606098; @raw:hanreplearn; @doi:10.1145/2806416.2806512], while other approaches can work directly with the edges themselves [@arxiv:1610.04073].
-We group these methods into the following three categories: matrix factorization, translational methods, and deep learning (Figure {@fig:unifying_techniques_overview}).
+The success of these approaches served as rationale for a sharper focus on representing knowledge graphs in a low dimensional space [@arxiv:1709.05584].
+This space captures the essence of a knowledge graph in the form of dense vectors, which are lists of continuous values of user-specified length; however, some algorithms pre-define the length of each vector [@doi:10.1007/BF02288367; @doi:10.1162/089976603321780317].
+These vectors are often assigned to nodes in a graph, but edges can be assigned as well.
```

Comment:
Cite at least one example from the literature for each to help your reader out or prune this sentence if you will discuss more fully in the section.